library/kotlin/src/io/envoyproxy/envoymobile/Request.kt

```diff
@@ -7,12 +7,14 @@ package io.envoyproxy.envoymobile
  * @param scheme The URL scheme for the request (i.e., "https").
  * @param authority The URL authority for the request (i.e., "api.foo.com").
  * @param path The URL path for the request (i.e., "/foo").
+ * @param upstreamHttpProtocol The protcol version to use for upstream requests.
  */
 data class Request internal constructor(
     val method: RequestMethod,
     val scheme: String,
     val authority: String,
     val path: String,
+    val upstreamHttpProtocol: UpstreamHttpProtocol,
```

Comment:
nit: I think we should prefer `HTTP` over `Http` on all of these, in line with how we use `MS` for milliseconds over `Ms`. wdyt?

Reply:
I feel conflicted because in the envoy codebase we dont use all caps...

Reply:
Eh just leave it I guess